src/libs/ErrorMessage/index.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
+import errorMessages from './errorMessages';
+
+/**
+ * It's generic function to print any messages.
```

Comment:
Let's remove the generic aspect of this. It's over-engineering. Right now we just need to handle error messages better in the context of `Authenticate`.

Reply:
Sure!

Reply:
Done, You can verify please.